_projects/2022/100348375/100348375.Rmd

```diff
@@ -0,0 +1,981 @@
+---
+title: "Replication and alternative visualization of a New York Times market share 
+graph"
+description: |
+ In this article I explain how I replicated a New York Times chart in R and the 
+ process of creating a different chart with ggplot2 with the same data. I'll 
+ also explain the limitations of replication and alternative visualization, and
+ discuss the strengths and weaknesses of both charts, based on data 
+ visualization theory.
+categories: "2022"
+author: Isabel Molero Lopez
+date: 2022-11-16
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+## **First step: replicate the graph**
```

Comment:
Please, do not set bold font in any title, because they have their own styling via CSS.